src/utils/traffic-mesh.js

```diff
@@ -10,6 +10,8 @@ const EXEC_NAME = 'traffic-mesh'
 
 const LATEST_VERSION = 'v0.22.1'
```

Comment:
We would need to update this too right?

Reply:
Yes. There's no release of mesh-forward with the new features yet though. I'll post a PR here once that's ready.